test/processors/feedback_url_swapper_test.rb

```diff
@@ -129,4 +129,28 @@ def test_should_cope_with_a_ascii_encoded_url
     assert_in template, "#test-input[value='/test?áscii=%EE%90%80']"
     assert_in template, "#test-input-two[value='https://example.com/test?áscii=%EE%90%80']"
   end
+
+  def test_should_cope_with_no_selector_being_found
+    template = as_nokogiri %(
+      <div>
+        <input
+          id="test-input"
+          name="NOT-email_survey_signup[survey_source]"
+          value="/old_path"
+        >
+        <input
+          id="test-input-two"
+          name="NOT-url"
+          value="https://example.com/old_path"
+        >
+      </div>
+    )
+
+    env = Rack::MockRequest.env_for("https://new-example.com/new_path")
+    request = Rack::Request.new(env)
+
+    headers = { Slimmer::Headers::TEMPLATE_HEADER => "gem_layout" }
+
+    Slimmer::Processors::FeedbackURLSwapper.new(request, headers).filter(nil, template)
```

Comment:
Super thanks Ian.

It'd be good to have an assertion in this test. A rather contrived one I've got is that the HTML doesn't change:

```suggestion
    original_html = template.to_s

    Slimmer::Processors::FeedbackURLSwapper.new(request, headers).filter(nil, template)
    
    assert_equal original_html, template.to_s
```

Reply:
Since we're testing that this doesn't throw an error, that seems okay - I've update the test. Feels as if this is edging closer to a `does_not_throw` oddity of a test!